runner/src/scenario/ceramic/event_id_sync.rs

```diff
@@ -0,0 +1,182 @@
+                tracing::error!("failed to setup model: {:?}", e);
+                return Err(e);
+            }
+        };
+        set_model_id(&mut conn, &model_id, MODEL_ID_KEY).await;
+        model_id
+    } else {
+        get_model_id(&mut conn, MODEL_ID_KEY).await
+    };
+
+    tracing::debug!(%model_id, "syncing model");
+
+    let path = format!("/ceramic/subscribe/model/{}?limit=1", model_id);
+    let user_data = ModelReuseLoadTestUserData {
+        cli,
+        redis_cli,
+        model_id,
+    };
+    user.set_session_data(user_data);
+    user.base_url = Some(ipfs_peer_addr); // Recon is only available on IPFS address right now
+
+    let request_builder = user
+        .get_request_builder(&GooseMethod::Get, &path)?
+        .timeout(Duration::from_secs(5));
+    let req = GooseRequest::builder()
+        .set_request_builder(request_builder)
+        .expect_status_code(200)
+        .build();
+
+    let _goose = user.request(req).await?;
+
+    Ok(())
+}
+
+/// Generate a random event that the nodes are interested in. Only one node should create but all
+/// users do it so that we can generate a lot of events.
+async fn create_new_event(user: &mut GooseUser) -> TransactionResult {
+    if !should_request_events() {
+        tokio::time::sleep(Duration::from_millis(500)).await;
+        Ok(())
+    } else {
+        let user_data: &ModelReuseLoadTestUserData = user
+            .get_session_data()
+            .expect("we are missing sync_event_id user data");
+
+        // eventId needs to be a multibase encoded string for the API to accept it
+        let event_id = format!("F{}", random_event_id(&user_data.model_id.to_string()));
+        let event_key_body = serde_json::json!({"eventId": event_id});
+        let cnt = NEW_EVENT_CNT.fetch_add(1, std::sync::atomic::Ordering::Relaxed);
+
+        if cnt == 0 || cnt % 1000 == 0 {
+            tracing::trace!("new sync_event_id body: {:?}", event_key_body);
+        }
+
+        let request_builder = user
+            .get_request_builder(&GooseMethod::Post, "/ceramic/events")?
+            .timeout(Duration::from_secs(1))
+            .json(&event_key_body);
+        let req = GooseRequest::builder()
+            .method(GooseMethod::Post)
+            .set_request_builder(request_builder)
+            .expect_status_code(204)
+            .build();
+        let _resp = user.request(req).await?;
+        Ok(())
+    }
+}
+
+fn random_cid() -> cid::Cid {
+    let mut data = [0u8; 8];
+    rand::Rng::fill(&mut rand::thread_rng(), &mut data);
+    let hash = Code::Sha2_256.digest(data.as_slice());
+    Cid::new_v1(0x00, hash)
+}
+
+const SORT_KEY: &str = "model";
+// hard code test controller in case we want to find/prune later
+const TEST_CONTROLLER: &str = "did:key:z6MkoFUppcKEVYTS8oVidrja94UoJTatNhnhxJRKF7NYPScS";
+
+fn random_event_id(sort_value: &str) -> ceramic_core::EventId {
+    let cid = random_cid();
+    EventId::new(
+        &ceramic_core::Network::Local(42),
```

Comment:
if this doesn't match the value of the configured nodes then it will not synchronize. I believe we use 0.

Reply:
Ah, good to know. Changed to 0. I wonder if this is was causing my lack of results.